packages/fetchai/connections/p2p_libp2p/aea/api.go

```diff
@@ -25,20 +25,22 @@ import (
 */
 
 type AeaApi struct {
-	msgin_path  string
-	msgout_path string
-	agent_addr  string
-	id          string
-	entry_peers []string
-	host        string
-	port        uint16
-	host_public string
-	port_public uint16
-	msgin       *os.File
-	msgout      *os.File
-	out_queue   chan *Envelope
-	closing     bool
-	sandbox     bool
+	msgin_path    string
+	msgout_path   string
+	agent_addr    string
+	id            string
+	entry_peers   []string
+	host          string
+	port          uint16
+	host_public   string
+	port_public   uint16
+	host_delegate string
+	port_delegate uint16
+	msgin         *os.File
+	msgout        *os.File
+	out_queue     chan *Envelope
+	closing       bool
+	sandbox       bool
 }
```

Comment:
Can we have comments somewhere here describing each attribute in more detail?

Reply:
You can reuse the ones from the python side

Reply:
will refactor that